base/io.jl

```diff
@@ -174,7 +174,12 @@ Read a single line of text, including a trailing newline character (if one is re
 the end of the input), from the given I/O stream or file (defaults to `STDIN`).
 When reading from a file, the text is assumed to be encoded in UTF-8.
 """
-readline(filename::AbstractString) = open(readline, filename)
+function readline(filename::AbstractString, chomp = false; nl2lf = false)
```

Comment:
Not the greatest keyword argument name. I wonder if it'd be better to do something like:
```
const LINEFEEDS = ['\n', '\r', '\u85', '\u0B', '\u0c', '\u2028', '\u2029']

function readline(file, chomp; newlines::Vector{Char}=linefeeds)
...
```
This has the dual-benefit of making the linefeeds vector a `const`, as well as allowing the user to specify a custom set of linefeed characters if desired.